exp-player/addon/components/exp-overview/component.js

```diff
@@ -107,13 +107,16 @@ export default ExpFrameBaseComponent.extend(Validations, {
     type: 'exp-overview',
     layout: layout,
     questions: questions,
-    responses: Ember.computed('questions', function() {
-        var questions = this.get('questions');
-        var responses = {};
-        for (var i=0; i < questions.length; i++) {
-            responses[i] = questions[i].value;
-        }
-        return responses;
+    showOptional: Ember.computed('questions.7.value', function() {
```

Comment:
As long as this works I'm fine with it-- I've just never seen an example of registering a computed on a specific array element like this. 

Looking at https://guides.emberjs.com/v2.6.0/object-model/computed-properties-and-aggregate-data/, I don't see this documented anywhere